.github/workflows/ci.yml

```diff
@@ -301,5 +301,4 @@ jobs:
         uses: actions/checkout@v4
       - name: Test if the documentation will render without warnings
         run: |
-          && docker run --rm --pull always -v $(pwd):/project \
-             ghcr.io/typo3-documentation/render-guides:latest --config=Documentation --no-progress --fail-on-log
+          composer docs:generate
```

Comment:
In cases where there's only one line for `run`, we can also drop the `|` notation and write the command directly after `run:`.

Reply:
Done, personally, I like it with the "|" as it is better readable IMHO.

Reply:
I'm fine either way (even if I prefer the one-line version). So please go ahead and use the version that you like best. :-)

Reply:
The composer command won't work, as it uses an interactive shell, which GitHub Actions does not like. Therefore the original command was set in place again.